crates/cxx-qt-build/src/lib.rs

```diff
@@ -499,6 +499,7 @@ impl CxxQtBuilder {
             builder.flag_if_supported("/permissive-");
```

Comment:
can you check with MSVC as well, do we need to enable `/bigobj` there for debug builds?

https://learn.microsoft.com/en-us/cpp/build/reference/bigobj-increase-number-of-sections-in-dot-obj-file?view=msvc-170

Reply:
Or maybe this isn't a problem yet there, just curious how we are hitting it in MinGW debug builds.

Reply:
I don't find any drawback to add it by default for MSVC. It only increases slightly the compilation time and can break some 3rd party debug tools if they don't support the biobj format

Reply:
Better be safe than sorry. I pushed the change